### ISSUE TITLE
What does GitHub do?

### ISSUE THREAD

**Issue by IAmSpecs** [Scratch-Client-4/landing#23]
Hello GitHubers ,( I dont know what to call you)  I heard about Scratch Client 4 from ekmand via scratch. On Scratch my username/s are: @ILoveMyCabin10 , @-Uplift- @-Vexol-. I dont know what I do on GitHub and would like some help! :)
Thanks, IAmSpecs.

**Comment by rudrathegreat** [comment]
Hey @IAmSpecs !

Github is a place where you can share, update and store code. The Scratch-Client 4 project's main aim to redeisgn the Scratch user interface. We have got a few teams. The frontend team deals with web development. The backend focuses on the mobile app which we are developing. And finally, we have the design who are designing some stuff. I'm a part of the frontend team.

In this repo, you will find that there are various branches. The develop branch contains the latest developments we have made to the site. To edit the code, you simply click on the file which you want to edit, and then click the edit button. Unfortunately, this simple method has various downsides and is the most cumbersome technique and can make things confusing. Instead, fork this project (aka remix it!) and then you will have a separate copy of the project. You can do whatever the hell you want with your fork. Once you have made the changes you wanted to make in your fork and you are ready to merge your code with the code in this respository, then simply click the pull request button to create a request to merge the latest version. One of the team members will take a look at the code, see if it's ready for deployment and merge it. My fork of the repository contains the latest updates so you should probably check that out. You can also check out the develop branch of this respository to see the latest updates.

Feel free to ask me questions. I'll be happy to help!

@rudrathegreat 
Find me on scratch here - @sekhrirudra

**Comment by IAmSpecs** [comment]
I am a young boy so, I will try to understand it. :) Can you import code to scratch?
Thanks, @IAmSpecs

**Comment by rudrathegreat** [comment]
I am 13 so it's aight. We're all young here! This repository is using HTML, CSS and Javascript, so the code cannot be imported to Scratch. You need an Integrated Development Environment (IDE) to edit code. I use PyCharm (because I LOVE python) but you can use whatever you want. As long as the IDE understands HTML, CSS and Javascript, you should be good.

**Comment by rudrathegreat** [comment]
Sorry, I'm going to be a little inactive, I'm trying to fix a bug which I am encountering. It might take a while. Have a play around, read some documentation on Github, watch some videos, because if you want to be a godlike developer (I am not one of them XP) then you will have to use Github, it's just that simple

**Comment by rudrathegreat** [comment]
hey @IAmSpecs how's it going with all the documentation on Github, etc. Hope that is clearing up some questions you might have!

**Comment by IAmSpecs** [comment]
> hey @IAmSpecs how's it going with all the documentation on Github, etc. Hope that is clearing up some questions you might have!

Thanks, I am 10 so. I tried java, but most of the offline HTML editors are scams. So I came out of java! I did enjoy python for sometime but did not find it as fun as offline scratch 2.0. I may come back to one language, don’t know witch but yeh.

Thanks, @IAmSpecs

**Comment by rudrathegreat** [comment]
It's alright. Quick question, have you made websites using html and css before?

**Comment by IAmSpecs** [comment]
> It's alright. Quick question, have you made websites using html and css before?

No. I don’t really know what css is? Can you explain it to me. And I tried do html but I figured that it would take to long to type. I used to type well, but alas I can anymore.

**Comment by YOYITsM3M8** [comment]
@IAmSpecs css is the stylesheet for html, how it looks, where its position, etc.

**Comment by micahlt** [comment]
@IAmSpecs Hi!  Unfortunately, you don't seem to have the qualifications right now to contribute to the Scratch Client 4 language.  We're looking for users who have experience with Git and GitHub, HTML5, CSS, ES6 JavaScript, Vue.js, and NPM.

**Comment by IAmSpecs** [comment]
Fine with me. Though I do have experience with JavaScript. I just came here to check out git. I don’t mind what I am on it’s what I will do.

**Comment by Cyan01-Hash** [comment]
Hello. Do you currently need help? If not you can ask me later any time. I have programmed with html and a little bit of css. (I usually use ejs)

**Comment by CompetitiveGaming** [comment]
> Hello. Do you currently need help? If not you can ask me later any time. I have programmed with html and a little bit of css. (I usually use ejs)

This is my other account btw. My previous one i couldn't get into. Do you do scratch?

**Comment by CompetitiveGaming** [comment]
> @IAmSpecs Hi! Unfortunately, you don't seem to have the qualifications right now to contribute to the Scratch Client 4 language. We're looking for users who have experience with Git and GitHub, HTML5, CSS, ES6 JavaScript, Vue.js, and NPM.

Ok. (My other account).
